src/net.cpp

```diff
@@ -41,8 +41,11 @@
 
 #include <math.h>
 
-// Dump addresses to peers.dat and banlist.dat every 15 minutes (900s)
-#define DUMP_ADDRESSES_INTERVAL 900
+// Dump addresses to peers.dat every 15 minutes (900s)
+static constexpr int DUMP_PEERS_INTERVAL = 60 * 15;
```

Comment:
Nit: `15 * 60` to be consistent with other `_INTERVAL` constants.

Reply:
Done!